bqetl_project.yaml

```diff
@@ -499,28 +500,45 @@ retention_exclusion_list:
 - sql/moz-fx-data-shared-prod/org_mozilla_tv_firefox_derived/baseline_clients_first_seen_v1
 - sql/moz-fx-data-shared-prod/org_mozilla_vrbrowser_derived/baseline_clients_first_seen_v1
 - sql/moz-fx-data-shared-prod/pine_derived/baseline_clients_first_seen_v11
-- sql/moz-fx-data-shared-prod/revenue_derived/client_ltv_v1
 - sql/moz-fx-data-shared-prod/telemetry_derived/core_clients_first_seen_v1
 - sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2
 - sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v1
 - sql/moz-fx-data-shared-prod/thunderbird_desktop_derived/baseline_clients_first_seen_v1
+- sql/moz-fx-data-shared-prod/accounts_backend_derived/users_services_first_seen_v1
+- sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_first_seen_v1
+- sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_first_seen_v2
+- sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_first_seen_v1
+- sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1
+- sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v2
+# mobile attribution tables
 - sql/moz-fx-data-shared-prod/fenix_derived/attribution_clients_v1
 - sql/moz-fx-data-shared-prod/firefox_ios_derived/attribution_clients_v1
 - sql/moz-fx-data-shared-prod/focus_android_derived/attribution_clients_v1
 - sql/moz-fx-data-shared-prod/focus_ios_derived/attribution_clients_v1
 - sql/moz-fx-data-shared-prod/klar_android_derived/attribution_clients_v1
 - sql/moz-fx-data-shared-prod/klar_ios_derived/attribution_clients_v1
+#
 - sql/moz-fx-data-shared-prod/firefox_desktop_derived/adclick_history_v1
 - sql/moz-fx-data-shared-prod/fenix_derived/client_adclicks_history_v1
-- sql/moz-fx-data-shared-prod/fenix_derived/clients_yearly_v1
```

Comment:
These are not being removed, just reschufling the order in the list to have a logic grouping